cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/analyzer/ExcludedBrokersForReplicaMoveTest.java

```diff
@@ -162,30 +165,30 @@
                                                          CpuUsageDistributionGoal.class)) {
       // Test: With single excluded broker, balance not satisfiable cluster, no dead broker (No exception, No proposal
       // for excluded broker, Not expected to look optimized)
```

Comment:
I think we should change the comment to be `Expected to look optimized` (drop the "Not")